api/utils.py

```diff
@@ -63,21 +63,24 @@ def update_config(config_dict, reboot_processor):
         return success
     return True
 
+def bad_request_serializer(msg, error_type="unknown error", loc=[]):
+    return [{
+        "loc": loc,
+        "msg": msg,
+        "type": error_type
+    }]
 
 def handle_response(response, success, status_code=status.HTTP_200_OK):
     if not success:
-        return JSONResponse(
+        raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
Taking into account that we are not sure about the reason of the error i think that a status 500 is better.
What do you think?

Reply:
Hmm but it wasn't exactly an error due to server issues right?

Reply:
A bad request error means that the user sent something wrong in the request; for example, he forgot to send a mandatory field. All these errors should be captured by the pydantic models (before call this function).

If we arrive into this function with an error means that for any reason (that we don't know) the server couldn't be restarted. Is common to return a 500 when you have an unexpected error in your system.